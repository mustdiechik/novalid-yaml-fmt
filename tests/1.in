
basicAuth:
  username: user
  password: name
  foo:
    - one
    - two
  bar:
  - one
  - two
  else:
    one:
            two: two
                        three:tree
----
one:
            two:
                            thee: end
---
sometext: |
            first
                    ---
                    one
                        two
                            thee
                    four
            five
    foo: bar
---
