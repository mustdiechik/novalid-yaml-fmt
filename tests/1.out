
basicAuth:
  username: user
  password: name
  foo:
    - one
    - two
  else:
    one:
      two: two
        three:tree
----
one:
  two:
    thee: end
---
sometext: |
                    ---
                    one
                        two
                            thee
                    four
            five
        foo: bar
---
